R/geom_swim_marker.R

```diff
@@ -13,122 +13,57 @@
 geom_swim_marker <- function(mapping = NULL, data = NULL,
                              stat = "identity", position = "identity",
                              ...,
-                             color = NULL,
-                             glyph = NULL,
-                             parse = FALSE,
-                             nudge_x = 0,
-                             nudge_y = 0,
                              check_overlap = FALSE,
                              size.unit = "mm",
                              na.rm = FALSE,
                              show.legend = NA,
                              inherit.aes = TRUE) {
-  # Copied from ggplot2::geom_text
-  if (!missing(nudge_x) || !missing(nudge_y)) {
-    if (!missing(position)) {
-      cli::cli_abort(c(
-        "Both {.arg position} and {.arg nudge_x}/{.arg nudge_y} are supplied.",
-        "i" = "Only use one approach to alter the position."
-      ))
-    }
-
-    position <- position_nudge(nudge_x, nudge_y)
-  }
```

Comment:
No need for all this nudging setup. Users can just use `position_nudge()` if they want.